packages/ember-data/lib/system/store.js

```diff
@@ -300,6 +300,17 @@ Store = Service.extend({
     });
     ```
 
+    To create a new instance of `App.Post` that has a relationship with `App.User`
```

Comment:
I think we want to move away from references to globals Ember Apps in the API docs (`App.foo`). Do you mind changing this line to "To create a new instance of a `Post` that has a relationship with a `User` record"? and updating line 295 to say " To create a new instance of a `Post`:"